magento-research/pwa-studio
# Fix ApolloContext

## Description

The latest `react-apollo` update broke our usage of `ApolloContext`.

- Now that it's a top-level export, we need to import it directly from `react-apollo`.
- Since the context value has changed, we need to destructure it to obtain `client`.
- Peregrine's version of `react-apollo` needs to match Venia's.

## Related Issue

Closes #1465.

## Verification Steps

1. `yarn install`
2. `yarn watch:venia`
3. Search for a product.

There should be no errors from the build or in the browser console.

## Proposed Labels for Change Type/Package


## Checklist:
